pkg/descheduler/policyconfig_test.go

```diff
@@ -862,6 +862,79 @@ strategies:
 				},
 			},
 		},
+		{
+			description: "v1alpha1 to internal with priorityThreshold",
+			policy: []byte(`apiVersion: "descheduler/v1alpha1"
+kind: "DeschedulerPolicy"
+strategies:
+  "PodLifeTime":
+    enabled: true
+    params:
+      podLifeTime:
+        maxPodLifeTimeSeconds: 5
+      namespaces:
+        include:
+          - "testleaderelection-a"
+      thresholdPriority: null
+      thresholdPriorityClassName: prioritym
+`),
+			result: &api.DeschedulerPolicy{
+				Profiles: []api.DeschedulerProfile{
+					{
+						Name: fmt.Sprintf("strategy-%s-profile", podlifetime.PluginName),
+						PluginConfigs: []api.PluginConfig{
+							{
+								Name: "DefaultEvictor",
+								Args: &defaultevictor.DefaultEvictorArgs{
+									PriorityThreshold: &api.PriorityThreshold{
+										Value: utilpointer.Int32(0),
```

Comment:
this is just `0` because in the unit tests there is no `prioritym` class to parse right? In a cluster this would be a real number?